src/docker/compose.dss-core.yaml

```diff
@@ -43,9 +45,11 @@ services:
       PGRST_DB_URI: "postgres://econia:econia@postgres:5432/econia"
       PGRST_DB_ANON_ROLE: web_anon
       PGRST_DB_SCHEMA: api
+      PGRST_DB_MAX_ROWS: 500
```

Comment:
How about adding to `example.env` a variable `POSTGREST_MAX_ROWS`?

Then, in this file you can use `PGRST_DB_MAX_ROWS: ${POSTGREST_MAX_ROWS}`

That will allow DSS users to set all configs in their local copy of `example.env`